### ISSUE TITLE
Cannot be resolved through 1.1.1.1

### ISSUE THREAD

**Issue by samimb** [melpa/melpa#6648]
Not sure if this is the right forum for this issue.
Could not find a place to report through Cloudflare. 

Melpa can no longer be resolved through Cloudflare DNS 1.1.1.1

**Comment by Silex** [comment]
``` shell
$ host melpa.org 8.8.8.8
Using domain server:
Name: 8.8.8.8
Address: 8.8.8.8#53
Aliases: 

melpa.org has address 178.128.185.1
melpa.org mail is handled by 10 mx.hover.com.cust.hostedemail.com.

$ host melpa.org 1.1.1.1
Using domain server:
Name: 1.1.1.1
Address: 1.1.1.1#53
Aliases: 

Host melpa.org not found: 2(SERVFAIL)
```

Well, obviously you'd ask Cloudfare, as other DNS are working just fine. Topics like https://community.cloudflare.com/t/dns-update-not-updating/37179 makes me thing Cloudflare DNS is not acting like a normal DNS, but you need to force-update it, etc.

**Comment by thomasf** [comment]
@silex the page you are linking to is about using cloudflare as a dns service, I don't think it has anything to do with this particular issue.

`whois melpa.org` says that it's using hoover as a dns service and they have an active notice about this right now https://hoverstatus.com/

**Comment by thomasf** [comment]
melpa.org does resolve through 1.1.1.1 right now for me right now.

**Comment by Silex** [comment]
I confirm it resolves fine now with 1.1.1.1 as well.
